README.md

```diff
@@ -34,7 +34,7 @@ How to install and run Brackets
 -------------------------------
 #### Download
 
-Installers for the latest stable build for Mac, Windows and Linux (Debian/Ubuntu) can be [downloaded here](http://brackets.io/).
+Installers for the latest stable build for Mac, Windows and Linux (Debian/Ubuntu) can be [downloaded here](https://brackets-cont.github.io/).
```

Comment:
this might not be necessary as the owner of the domain has popped up in matrix :-)

let's keep on eye out on how that develops and maybe wait for few days before merging this

Reply:
A reason for waiting with the domain-swap was brought up in Matrix:
The official EOL is until September. Let's merge this :-)